template-only-bin/destroy-app-service.sh

```diff
@@ -7,11 +7,12 @@ BACKEND_CONFIG_FILE="dev.s3.tfbackend"
 
 sed -i.bak 's/force_destroy = false/force_destroy = true/g' infra/modules/service/access-logs.tf
 sed -i.bak 's/force_destroy = false/force_destroy = true/g' infra/modules/storage/main.tf
+sed -i.bak 's/enable_deletion_protection = var.is_temp_environment/enable_deletion_protection = false/g' infra/modules/service/load-balancer.tf
```

Comment:
i think you just copy pasted my snippet, but you have to make sure that the line you are replacing is the same as the actual line. looks like you named the variable is_temporary rather than is_temp_environment

Reply:
🤦